index.html

```diff
@@ -12,94 +12,124 @@
+					<div class="dash days_dash col-3 col">
+						<div class="digits clearfix">
+							<div class="digit digit-1">0</div>
+							<div class="digit digit-2">0</div>
+							<div class="digit digit-3">0</div>
+						</div>
+						<div class="dash_title"><span>days</span></div>
 					</div>
-					<div class="dash_title"><span>days</span></div>
-				</div>
-				<!-- ## HOURS ## -->
-				<div class="dash hours_dash col-3 col">
-					<div class="digits clearfix">
-						<div class="digit digit-1">0</div>
-						<div class="digit digit-2">0</div>
+					<!-- ## HOURS ## -->
+					<div class="dash hours_dash col-3 col">
+						<div class="digits clearfix">
+							<div class="digit digit-1">0</div>
+							<div class="digit digit-2">0</div>
+						</div>
+						<div class="dash_title"><span>hours</span></div>
 					</div>
-					<div class="dash_title"><span>hours</span></div>
-				</div>
-				<!-- ## MINUTES ## -->
-				<div class="dash minutes_dash col-3 col">
-					<div class="digits clearfix">
-						<div class="digit digit-1">0</div>
-						<div class="digit digit-2">0</div>
+					<!-- ## MINUTES ## -->
+					<div class="dash minutes_dash col-3 col">
+						<div class="digits clearfix">
+							<div class="digit digit-1">0</div>
+							<div class="digit digit-2">0</div>
+						</div>
+						<div class="dash_title"><span>minutes</span></div>
 					</div>
-					<div class="dash_title"><span>minutes</span></div>
-				</div>
-				<!-- ## SECONDS ## -->
-				<div class="dash seconds_dash col-3 col">
-					<div class="digits clearfix">
-						<div class="digit digit-1">0</div>
-						<div class="digit digit-2">0</div>
+					<!-- ## SECONDS ## -->
+					<div class="dash seconds_dash col-3 col">
+						<div class="digits clearfix">
+							<div class="digit digit-1">0</div>
+							<div class="digit digit-2">0</div>
+						</div>
+						<div class="dash_title"><span>seconds</span></div>
 					</div>
-					<div class="dash_title"><span>seconds</span></div>
 				</div>
+				<!-- ## COUNTDOWN TIMER END ## -->
+				<a id="slidelink" href="https://ti.to/write-speak-code/2017-conference/" target="_blank" class="qcTargetLink video">Buy Tickets</a>
 			</div>
-			<!-- ## COUNTDOWN TIMER END ## -->
-
-			<a id="slidelink" href="https://ti.to/write-speak-code/2017-conference/" target="_blank" class="qcTargetLink video">Buy Tickets</a>
-
 		</div>
-
 	</section>
-	<!-- ## FULLSCREEN SLIDES ## -->
 
+	<!-- ## SPONSOR SECTION ## -->
+	<section class="qcContainer sponsors-section" id="sponsors">
+	  <div class="row middle-md">
+	    <div class="col-xs-12 col-md-6">
+	      <h2>Our Sponsors</h2>
+	    </div>
+	    <div class="col-xs-12 col-md-6 end-md">
+	      <a href="sponsor-prospectus.pdf" target="_blank" class="prospectus-link">Interested in becoming a sponsor?</a>
+	    </div>
+	  </div>
+	  <div class="row sponsor-level">
+	    <div class="col-xs-12">
+	      <h3>Silver Level Sponsors</h3>
+	    </div>
+	    <div class="col-xs-12 col-sm-4">
```

Comment:
There is an easy way to center! You add a class to the row container of "center-xs" or -sm or -md or -lg depending on what breakpoint you want the row to be centered.

However, aesthetically I'm not sure it looks better centered because there is still awkward space around the sides. 
![screen shot 2017-02-20 at 10 00 22 am](https://cloud.githubusercontent.com/assets/11186572/23130228/9bcb1ed8-f753-11e6-8614-6c43546492bc.png)

I'm wondering if a better solution would be to add placeholder text/images/boxes that say "Your logo here! etc" and maybe subtly link to the prospectus? What do you think?